### PR TITLE
rename restic-init-container to ark-restic-restore-helper

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <steve@heptio.com>

### CODE REVIEW

File: pkg/restore/restic_restore_action.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func (a *resticRestoreAction) Execute(obj runtime.Unstructured, restore *api.Res
 
 	initContainer := corev1.Container{
 		Name:  restic.InitContainer,
-		Image: "gcr.io/heptio-images/restic-init-container:latest",
+		Image: "gcr.io/heptio-images/ark-restic-restore-helper:latest",
```
[the remark this answers is not in the document]
Comment (skriss) [reply pkg/restore/restic_restore_action.go]: TODO'ed

**Comment by ncdc** [comment]
Actually since we aren't going to be setting `latest` for `v0.9.0.alpha.1`, we may need to handle that TODO now. WDYT?

**Comment by skriss** [comment]
can we just grab buildinfo.Version? That's an easy change

**Comment by ncdc** [comment]
That helps with the tag but not the rest of the image URL. But it's a good start, so let's do it. And have the cli flag before we cut 0.9 final.

File: pkg/restore/restic_restore_action.go [hunk 2]
```diff
@@ -25,20 +27,32 @@ import (
...
 	}
 }
 
+func initContainerImage() string {
+	tag := buildinfo.Version
+	if tag == "" {
+		tag = "latest"
+	}
+
+	return fmt.Sprintf("gcr.io/heptio-images/ark-restic-restore-helper:%s", tag)
```
Comment (ncdc) [inline pkg/restore/restic_restore_action.go]: You lost the TODO

Comment (skriss) [reply pkg/restore/restic_restore_action.go]: urgh. cut and failed to paste :/

Comment (skriss) [reply pkg/restore/restic_restore_action.go]: upd.

**Comment by ncdc** [comment]
LGTM

**Pull request merged**